src/login.js

```diff
@@ -73,6 +76,8 @@ export async function login({
     case "linkedin":
     case "okta":
       return signonWithSso({ provider: method, redirect });
+    case "custom sso":
+      return signonWithSso({ provider: method, redirect, providerId });
```

Comment:
Wasn't sure whether to whether to go with a space or `-` here. It looks like we could use a single convention for the `method`s:
- `custom sso`
- `password-migrate`
- `verificationCode`

Reply:
Could we get away with just `custom` here? It would be much cleaner. Also , consider `password-migrate` as temporary. It only does anything for Pay.com.au

Reply:
That's what I had it named originally then changed it after looking at the list in the [docs](https://userfront.com/docs/js#login-options) to make it more clear. If you don't foresee any confusion with `custom` we can go with that.

Reply:
I don't foresee any more issues with `custom` than with `custom sso`